server/sources.py

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/python3
+import igraph
+import io
+import json
+from lazy_property import LazyProperty
+import numpy as np
+# import psycopg2
```

Comment:
Is this import necessary?

Reply:
Removed the `os` one. Psycopg2 is just used for StringDB. I'll remove it as you propose to replace it with aiopg anyway.